README-dev.md

```diff
@@ -94,7 +94,7 @@ For a shorter feedback loop, you can run the mpyl sourcecode against another rep
 To test the mpyl sourcecode against the peculiarities of your own repository, you can run the following command:
 
 ```shell
-PIPENV_PIPFILE=/<absolute_path_to_mpyl_repo>/Pipfile pipenv run cli-ext build status
+PIPENV_PIPFILE=/<absolute_path_to_mpyl_repo>/Pipfile pipenv run cli build status
```

Comment:
So this is what I meant in the other comment, this doesn't work with the `cli` command. It's what `cli-ext` was made for